inst/extdata/sitevisitpeak-example.json

```diff
@@ -51,6 +51,7 @@
     "altitudeDatumCode": "NGVD29    ",
     "siteNumber": "07105490       ",
     "stationName": "CHEYENNE CREEK AT EVANS AVE AT COLORADO SPRINGS,CO",
+    "sublocation": "Sub-Location",
```

Comment:
@dpattermann-usgs is this right? (Minus the baloney example of Sub-Location...?)